src/Forms/Shared/Samples/Geometry/NearestVertex/NearestVertex.xaml.cs

```diff
@@ -41,38 +42,48 @@ public NearestVertex()
 
         private void Initialize()
         {
-            // Configure the basemap
-            MyMapView.Map = new Map(BasemapStyle.ArcGISTopographic);
+            // Create a spatial reference using the California zone 5 (ftUS) state plane coordinate system.
+            SpatialReference californiaZone5SpatialReference = SpatialReference.Create(2229);
 
-            // Create the graphics overlay and set the selection color
+            // Create a map that uses the California zone 5 state spatial reference.
+            MyMapView.Map = new Map(californiaZone5SpatialReference);
+
+            // Create the feature layer. 
+            Uri uriLayerSource = new Uri("https://services.arcgis.com/P3ePLMYs2RVChkJx/arcgis/rest/services/USA_States_Generalized/FeatureServer/0");
+            FeatureLayer usaStatesFeatureLayer = new FeatureLayer(uriLayerSource);
+
+            // Add the feature layer to the MapView.
+            MyMapView.Map.OperationalLayers.Add(usaStatesFeatureLayer);
+
+            // Create the graphics overlay and set the selection color.
             _graphicsOverlay = new GraphicsOverlay();
 
-            // Add the overlay to the MapView
+            // Add the overlay to the MapView.
             MyMapView.GraphicsOverlays.Add(_graphicsOverlay);
 
-            // Create the point collection that defines the polygon
-            PointCollection polygonPoints = new PointCollection(SpatialReferences.WebMercator)
+            // Create the point collection that defines the polygon.
```

Comment:
This is an unusual spatial reference choice for the sample. My understanding is that this was deliberate, so that when the planar distance calculation is done later, the results will accurately reflect the real-world distance? That should be mentioned in the comment here.